README.md

```diff
@@ -1,3 +1,28 @@
+Leaflet was created 11 years ago by [Vladimir Agafonkin](https://agafonkin.com/), an Ukrainian citizen living in Kyiv.
+
+Vladimir is no longer in Kyiv, because Russian bombs are falling over the city. His family, his friends, his neighbours, thousands and thousands of absolutely wonderful people, are either seeking refuge or fighting for their lives.
+
+The Russian occupation of Ukraine contravenes and violates several international laws, including the United Nations Charter. It is an unprovoked, unjustifiable, and despicable action that endangers countless civilians.
+
+As Vladimir [expressed a few days ago](https://twitter.com/LeafletJS/status/1496051256409919489):
+
+> If you want to help, educate yourself and others on the Russian threat, follow reputable journalists, demand severe Russian sanctions and Ukrainian support from your leaders, protest war, reach out to Ukrainian friends, donate to Ukrainian charities. Just don't be silent.
+
+Ukrainians are recommending the [https://savelife.in.ua/en/donate/](https://savelife.in.ua/en/donate/) charity. Consider also the [Ukrainian Red Cross](https://redcross.org.ua/en/).
+
+If an appeal to humanity doesn't work for you, I'll appeal to your egoism: the future of Ukraininan citizens is the future of Leaflet.

```

Comment:
Minor typo "Ukraininan" should be "Ukrainian"

Reply:
Thanks for noticing! I just force-pushed the typo fix to main: 3bf06774